### DISCUSSION TITLE
Taiko stage "nubs"

### DISCUSSION THREAD

**Discussion by GamingWithJanko** [ppy/osu#25183] [Q&A] [answered]
Peppy, remember the mania side nubs*?

As a multi-mode player I noticed, that they weren't removed in all scrollbar modes.

In Taiko they're still there, which I don't play as often as mania. I thought you removed them globally and replaced them with the accented lines, but that was a mania-only thing it seems.

Are there any plans for taiko to undergo the same change in the future?

*These white "Pellets" next to the stage.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/25222
